### DISCUSSION TITLE
2022年：直播分析简历和求职问题

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/Interview#352] [Q&A]
大家可以留言自己的困惑【具体的公司或者姓名，记得打码】

时间预计在 19:30分左右【因为我19点下班】（记得提前下载一个“腾讯会议”）

**Comment by wantsieep** [comment]


**Reply by wj-Mcat** [reply]
不建议，因为这个对理论要求比较高，即时是调参侠也是需要看论文，掌握最新模型技术动态。

而且现在形势不好，大厂hc 收缩，导致供大于求，转ai 竞争会非常大。

所以不太建议。

**Comment by jiangzhonglian** [comment]
2022-07-02 #腾讯会议：635-817-631

**Comment by kangyuqi** [comment]
已收到您的邮件，谢谢。

**Comment by jiangzhonglian** [comment]
2022-07-17 #腾讯会议：416-330-458

**Comment by kangyuqi** [comment]
已收到您的邮件，谢谢。

**Comment by kangyuqi** [comment]
已收到您的邮件，谢谢。
